### PR TITLE
Added navbar in booklist.html

### PR DESCRIPTION
**Pull request by kanishka8276**
# Description
Added navbar in booklist.html and did styling of the list of books. Also linked about under useful links correctly.
Fixes # (issue)
#371
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Screenshot (238)

### CODE REVIEW

**Review by ravikr126 — APPROVED** [review]

**Review by ravikr126 — CHANGES_REQUESTED** [review]
add books pdfs

File: booklist.html [hunk 1]
```diff
@@ -31,8 +31,82 @@
...
+                        </div>
+                        <li class="nav-item">
+                        <a class="nav-link  font-weight-bold nav-items" href="#footer">Contact</a>
+                    </li>
+                    </ul>
+
+            </div>
+        </nav>
+        <script>
+            const navbar=document.getElementById("nav");
```
Comment (harikesh409) [inline booklist.html:81]: Add the `nav` id to the nav tag.

File: booklist.html [hunk 2]
```diff
@@ -31,8 +31,82 @@
...
+            let scrolled=false;
+            window.onscroll=function(){
+                if(window.pageYOffset>100)
+                {
+                    navbar.classList.remove("top");
```
Comment (harikesh409) [inline booklist.html:86]: Include the `top` class to the navbar.

File: booklist.html [hunk 3]
```diff
@@ -31,8 +31,82 @@
...
+                    },200);
+                }
+                else{
+                    navbar.classList.add("top");
+                    scrolled=false;
+                }
+            };
+            </script>
+    </section>
+    <center>
```
Comment (harikesh409) [inline booklist.html:103]: `<center>` tag is deprecated so don't use it.

Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by kanishka8276** [comment]
@harikesh409 Made the suggested changes. Please review.
Screenshot (253)

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Also the social icons style in follow us of footer is different from home page change those also to the style as it is present in the home page.

**Review by harikesh409 — APPROVED** [review]

**Comment by ravikr126** [comment]
wait for some more time when pa give their drive access i will give you the links of pdf and then you add then we merge this PR.
@kanishka8276

**Comment by kanishka8276** [comment]
@ravikr126 Please merge this PR. I can create another PR for adding books as otherwise I will have to keep on resolving the conflicts.

**Comment by harikesh409** [comment]
> @ravikr126 Please merge this PR. I can create another PR for adding books as otherwise I will have to keep on resolving the conflicts.

Merge markers are present in the files remove those and push then I can merge it.

**Comment by ravikr126** [comment]
@kanishka8276

**Comment by kanishka8276** [comment]
@harikesh409 @ravikr126 Done!

**Pull request merged**